lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -435,6 +435,9 @@ mkDelegationCertificates da accXPub =
 -- `updateSealedTx`.  See `updateSealedTx` for more details.
 data ExtraTxBodyContent = ExtraTxBodyContent
     { extraInputs :: [TxIn]
+    , extraCollateral :: [TxIn]
+       -- ^ Only used in the Alonzo era and later. Will be silently ignored in
+       -- previous eras.
```

Comment:
Thought about `error`ing, but decided to ignore instead.

I don't think returning `Left` here is a good idea, as passing collateral to a Shelley tx would be a programmer and not user error.